src/views/ev3/l10n.json

```diff
@@ -18,12 +18,12 @@
     "ev3.clickMotorBlock": "Find the {motorBlockText} block and click on it.",
     "ev3.motorBlockText": "\"motor A turn this way\"",
     "ev3.starterProjects": "Starter Projects",
-    "ev3.waveHelloTitle": "Wave Hello",
-    "ev3.waveHelloDescription": "Make a puppet robot and have a friendly chat.",
-    "ev3.distanceInstrumentTitle": "Distance Instrument",
-    "ev3.distanceInstrumentDescription": "Move your body in front of the sensor to make music.",
-    "ev3.spaceTacosTitle": "Space Tacos",
-    "ev3.spaceTacosDescription": "Build your own controller to catch tacos in space.",
+    "ev3.starter1Title": "Play Basketball",
+    "ev3.starter1Description": "Move in front of the distance sensor to bounce the ball.",
+    "ev3.starter2Title": "Make Music",
+    "ev3.starter2Description": "Press the buttons to play saxophone and drums.",
+    "ev3.starter3Title": "Space Tacos",
+    "ev3.starter3Description": "Build your own controller to catch tacos in space.",
```

Comment:
Wondering why these keys are changing — I think it's better for the ID to be specific to the content so that we don't re-use the id unnecessarily. E.g. if we change what starter1 is just by updating the translation string, it's possible for the English string to say something else, but old translations will still say the previous content until the new translation is done and reviewed. In that case it would be better for us to display English but with the correct content.

Reply:
good point- updating the ids!